### ISSUE TITLE
Font not rendering properly with Linux Binary?

### ISSUE THREAD

**Issue by nicefellow1234** [wkhtmltopdf/wkhtmltopdf#3585]
I have just started using WKHTMLTOPDF and I really love it on Windows SERVER but on Linux it has issues for instance the font which I used in my HTML Template that is not rendered properly on Linux SERVER but is rendered properly on Windows SERVER.

Why would this be happening, I have no clue and my online SERVER is Linux based so I want to use the Linux Binary to render the font properly?

Please have a look over the results as : 
![image](https://user-images.githubusercontent.com/10282608/28822590-41622a7c-76d3-11e7-83b8-dcaca3629ab4.png)

**YOU CAN PERFECTLY SEE THAT THE FONTS RENDERING IS NOT IDENTITAL**

So how would I be able to achieve the same results as Windows on Linux SERVER too?

**Comment by PhilterPaper** [comment]
The problem with arcs in letters being replaced by chords (straight lines) has been discussed a number of times before, but I don't recall if anyone noted a difference in Linux vs Windows. Have you checked those bug reports (issues)? There _have_ been reports of different text size between the two operating systems. Regarding the overlap on the top line (Linux only), I don't recall hearing about that before. I _do_ notice that the font aspect ratio seems to be different between the two.

By the way, have you tried producing a PDF on Linux and viewing it on Windows? That would rule in (or out) that it's the PDF reader on Linux that's screwing up, and not wkHTMLtoPDF.

**Comment by nicefellow1234** [comment]
Yes I've generated PDF on both OS and there is a huge difference in fonts between both. And Yes I am checking both ones on Windows And I have checked the same PDF on different PDF Readers i.e on Chrome & on Windows Edge and on both it shows the same issues.
[WINDOWS EXECUTABLE CONVERTED.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/1190737/WINDOWS.EXECUTABLE.CONVERTED.pdf)
[LINUX BINARY CONVERTED.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/1190744/LINUX.BINARY.CONVERTED.pdf)

**Comment by megamindbrian** [comment]
Interesting.  Does the CSS     body{ text-rendering: geometricPrecision; }
make any difference?

**Comment by nicefellow1234** [comment]
@megamindbrian : I've checked it and it's the same as before no difference.

**Comment by elektro-wolle** [comment]
Have you set the fontconfig in `/etc/fonts/conf.d/10-wkhtmltopdf.conf `as mentioned in [https://github.com/wkhtmltopdf/wkhtmltopdf/issues/45#issuecomment-319442532](https://github.com/wkhtmltopdf/wkhtmltopdf/issues/45#issuecomment-319442532)?

**Comment by romalfc** [comment]
Have similar problem when embed GothamPro  font with `@font-face` css property. 
And after spending a lot of time, resolve it with `src: url('GothamPro.ttf')` to `src: url('GothamPro.svg') format("svg");`

**Comment by extrahour** [comment]
I'm having same issue by using Work Sans font on Linux. Changing format from woff2 to svg did solved the problem however file size increased dramatically: 100kb -> 8mb. On the one hand it makes sense as the letters are converted to SVGs on the other hand the fix is not really usable. Unless the PDF contains very few information.

**Comment by jason-medeiros** [comment]
Issue is happening to me as well with version 0.12.5.  Standard arial font is fine, but any google webfonts wind up rendering a bit off.

**Comment by cseelus** [comment]
Same issue here with 0.12.5, macOS is fine, on Linux (Heroku) arcs in letters are being replaced by chords.

Using the Muli font only `.woff` files work (although font appears bolder than it should and outlines are also not 100% perfect. Otherwise `.svg` looks perfect, but the generated PDF is 5-10 times larger than usual, which is a no-go for our purposes (sending reports via E-Mail).

**Comment by prabhakn** [comment]
Thank you for refering this issue. In windows its working fine

**Comment by godric-cz** [comment]
Converting font to `.otf` with https://onlinefontconverter.com/ and using `@font-face` with `src: url('WorkSans-Regular.otf')` fixed it for me.

**Comment by mutley** [comment]
Converting the font to an .otf worked for me as well. We are embedding the fonts in the .HTML file using this tool. https://www.font-converter.net/en/css-embedded-font-base64

**Comment by leabelzunces** [comment]
> Confirming this to be an issue with (at least) all of these upstream binaries on top of Debian stretch:
> 
> * [0.12.3](https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.3/wkhtmltox-0.12.3_linux-generic-amd64.tar.xz)
> * [0.12.4](https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz)
> * [0.12.5](https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.stretch_amd64.deb)

And also 0.12.6
